subt/breadcrumbs.py

```diff
@@ -4,18 +4,30 @@
 from ast import literal_eval
 
 from osgar.node import Node
+from subt.trace import distance3D
 
 
 class Breadcrumbs(Node):
     def __init__(self, config, bus):
         super().__init__(config, bus)
-        bus.register("deploy")
+        bus.register("deploy", "location")
 
         self.robot_name = config.get('robot_name')
         self.start_time = None  # unknown
         self.next_deploy_time = None
         self.step_time = config.get('step_sec')
 
+        self.radius = config.get('radius')
+        self.locations = [[0, 0, 0]]  # all locations + fake for the base
+
+    def should_deploy(self, xyz):
+        if self.radius is None:
```

Comment:
Oh, so not providing `radius` configuration is a way to turn breadcrumb deployment off?

Reply:
ACK